tests/_async/test_client.py

```diff
@@ -119,14 +121,18 @@ async def test_client_upload_file(
     file_name = file["name"]
     file_path = file["local_path"]
     mime_type = file["mime_type"]
+    file_content = file["file_content"]
     bucket_file_path = file["bucket_path"]
     bucket_folder = file["bucket_folder"]
     options = {"content-type": mime_type}
 
     await storage_file_client.upload(bucket_file_path, file_path, options)
+
+    sleep(3)
```

Comment:
Why is this not an `asyncio.sleep`?

Reply:
Hi! Since it has to be compatible with sync.